src/components/ContextualMenu/ContextualMenu.test.tsx

```diff
@@ -266,4 +267,24 @@ describe("ContextualMenu ", () => {
     );
     expect(screen.getByText("toggleLabel component text")).toBeInTheDocument();
   });
+
+  it("forwards close callback to children that are not links", async () => {
+    const ContextMenuContent: FC<{ close?: () => void }> = ({ close }) => (
```

Comment:
Only a very minor thing, but we don't typically use `FC` in this project (though I think the `FC` type is better in React 18). Not a big deal being as this is in a test, but this could just be `const ContextMenuContent = ({ close }: { close?: () => void }) => (`.